libprecompiled/Precompiled.h

```diff
@@ -80,8 +80,13 @@ class Precompiled : public std::enable_shared_from_this<Precompiled>
         return m_precompiledExecResultFactory;
     }
 
+    // for UT
+    static void clearName2SelectCache();
```

Comment:
if s_name2SelectCache is not static, this function can not static

Reply:
Done.